tensorbay/client/job.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+
+"""Basic structures of asynchronous jobs."""
+
+from typing import Any, Dict, Optional
+
+
+class GetLogResponse:
+    """This class defines the structure of the response from interface get_log."""
+
+    def __init__(self, logs: str) -> None:
+        self._logs = logs
+
+    def _get_logs(self, size: int = 100) -> str:
+        """Get some logs with limited size.
+
+        Arguments:
+            size: The limited size of log.
+
+        Return:
+            A string contains some logs.
+
+        """
+
+    def get_logs(self) -> str:
+        """Get some logs.
+
+        Return:
+            A string contains some logs.
+
+        """
+
+    def get_all_logs(self) -> str:
+        """Get all logs.
+
+        Return:
+            A string contains all logs.
+
+        """
+
+
+class Job:  # pylint: disable=too-many-instance-attributes
+    """This class defines :class:`Job`.
+
+    Arguments:
+        title: Title of the Job.
+        job_id: ID of the Job.
+        job_type: Type of the Job.
+        creator: The creator of the Job.
+        arguments: Arguments of the Job.
+        created_at: The time when the Job is created.
+        started_at: The time when the Job is started.
+        max_retries: The maximum retry times of the Job.
+        finished_at: The time when the Job is finished.
+        description: The description of the Job.
+
+    """
+
+    def __init__(  # pylint: disable=too-many-arguments
+        self,
+        title: str,
+        job_id: str,
+        job_type: str,
+        creator: str,
+        arguments: Dict[str, Any],
+        created_at: str,
+        started_at: str,
+        max_retries: int,
+        finished_at: Optional[str] = None,
+        description: Optional[str] = "",
+    ) -> None:
+        self.title = title
+        self.job_id = job_id
+        self.job_type = job_type
+        self.creator = creator
+        self.arguments = arguments
+        self.created_at = created_at
+        self.started_at = started_at
+        self.max_retries = max_retries
+        self.finished_at = finished_at
+        self.description = description
+
+    def get_status(self) -> Dict[str, Any]:
```

Comment:
what does the status mean?
from the doc seems `get_status` returns a dict which contains `status` + `code` + `errorMessage`?
which means `status` == `status + code + errorMessage`? I think it is weird.

Reply:
If the function name cause misunderstanding, we could use `get_status_info`.

Reply:
it is also weird to put `status + code + errorMessage` together?
when `status == SUCCESS` there are no `code` and `errorMessage`?
Shall we separate `get status` and `get error info`?